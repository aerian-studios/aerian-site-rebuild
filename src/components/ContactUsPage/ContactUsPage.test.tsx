/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { ContactUsPage } from "./index";

describe("ContactUsPage", () =>
    it("renders correctly", () => {
        const tree = renderer.create(<ContactUsPage />).toJSON();
        expect(tree).toMatchSnapshot();
    }));
