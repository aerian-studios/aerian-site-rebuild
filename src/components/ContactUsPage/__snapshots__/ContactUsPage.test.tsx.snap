// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsPage renders correctly 1`] = `
<section>
  <p
    className=" "
  >
    Loading
  </p>
  <div
    className=" "
  >
    <h2>
      ContactForm
    </h2>
  </div>
  <div>
    Facebook
  </div>
  <div>
    Twitter
  </div>
  <div>
    G+
  </div>
  <div>
    LinkedIn
  </div>
</section>
`;
